zerver/tests/test_service_bot_system.py

```diff
@@ -138,27 +145,30 @@ def test_invalid_calls(self):
         storage.marshal = lambda obj: obj
         storage.demarshal = lambda obj: obj
         serializable_obj = {'foo': 'bar', 'baz': [42, 'cux']}
-        with self.assertRaisesMessage(StateHandlerError, "Cannot set state. The value type is "
-                                                         "<class 'dict'>, but it should be str."):
+        with self.assertRaisesMessage(StateError, "Cannot set state. The value type is "
+                                                  "<class 'dict'>, but it should be str."):
             storage.put('some key', serializable_obj)  # type: ignore # We intend to test an invalid type.
-        with self.assertRaisesMessage(StateHandlerError, "Cannot set state. The key type is "
-                                                         "<class 'dict'>, but it should be str."):
+        with self.assertRaisesMessage(StateError, "Cannot set state. The key type is "
+                                                  "<class 'dict'>, but it should be str."):
             storage.put(serializable_obj, 'some value')  # type: ignore # We intend to test an invalid type.
 
     def test_storage_limit(self):
         # type: () -> None
         # Reduce maximal state size for faster test string construction.
-        StateHandler.state_size_limit = 100
+        settings.USER_STATE_SIZE_LIMIT = 100
```

Comment:
Changed this to use `@override_settings`, since that cleans up after itself at the end of the test (this doesn't!)